### PR TITLE
Entity resolution search now returns all phone numbers

### LINKED ISSUE #3: Fix entity search results
Entity search result data seems to disagree with results from manually querying the SQL database.

Tasks:
- [x] Identify source of the bug
- [ ] Make the fix

### PR DESCRIPTION
**Pull request by coxjonc**
Used nested pg queries to get all entity phone numbers as a list, instead of just returning the phone number given in the query.

We'll want to refactor this into something that makes use of a library like [pg-promise](https://github.com/vitaly-t/pg-promise) at some point, since nested callbacks are kind of an antipattern.

Fixes #3

### CODE REVIEW

File: routes/queries.js [hunk 1]
```diff
@@ -71,26 +81,49 @@ function QueryHandler(client) {
         } else if(req.query["phone"]) {
             phones.push(req.query["phone"]);
         }
-        console.log(phones);
-        console.log("match: " + (phones[0] == '678-680-9278'));
+
         q += 'backpageemail.name = ANY(\'{"' + emails.join('","') + '"}\'::text[]) OR backpagephone.number = ANY(\'{"' + phones.join('","') + '"}\'::text[])'
-        q += " GROUP BY backpageentities.entity_id;"
+
+        console.log(q);
```
Comment (dlrobertson) [inline routes/queries.js]: Debug statement :)

File: routes/queries.js [hunk 2]
```diff
@@ -71,26 +81,49 @@ function QueryHandler(client) {
...
-            console.log(q);
             if(err) {
                 console.error("error running query", err);
                 return res.status(500).json({
                     "message": err
                 });
             }
 
-            var data = { "entities": [] };
-            console.log(result.rows);
```
Comment (dlrobertson) [inline routes/queries.js:88]: Thanks

**Review by dlrobertson — CHANGES_REQUESTED** [review]
Looks good, but it looks like you missed a `console.log`. Have you ran and tested this locally?

**Comment by coxjonc** [comment]
Whoops - I'll go ahead and strip out the console logs :). I have tested it locally, though I'd recommend that you or Bryant play with it too before we merge to make sure there aren't any edge cases I missed.

**Comment by coxjonc** [comment]
Removed the console log - let me know if there's anything else that needs to be changed

**Review by dlrobertson — APPROVED** [review]

**Comment by dlrobertson** [comment]
Thanks for working on this!

**Pull request merged**